python/CMakeLists.txt

```diff
@@ -52,19 +56,19 @@ target_link_libraries(_pycudaq PUBLIC nvqir
       cudaq-platform-default 
       fmt::fmt-header-only)
 
-install(TARGETS _pycudaq DESTINATION ${CMAKE_INSTALL_PREFIX})
-install(FILES cudaq.py DESTINATION ${CMAKE_INSTALL_PREFIX})
+# install(TARGETS _pycudaq DESTINATION  ${CMAKE_INSTALL_PREFIX})
```

Comment:
I've commented out the installation of targets to the install and build directories as I sort out other pieces of the code